3rdparty/simlib/include/mata/simlib/util/shared_list.hh

```diff
@@ -108,7 +108,7 @@ public:
 		}
 
 		if (elem)
-			--elem->counter_;
+			--elem->refcount_;
```

Comment:
This seems like quite random fix, how did you find this bug?

Reply:
It simply does not compile. Maybe it has something to do with the emscripten that is used for the build (but the `counter_` is indeed not present there, it is a real error).